src/Surfnet/ServiceProviderDashboard/Application/Service/AttributeService.php

```diff
@@ -30,19 +30,56 @@ class AttributeService implements AttributeServiceInterface
 
     private $attributes = [];
 
-    public function __construct(AttributeRepository $attributeRepository)
+    /**
+     * @var string
+     */
+    private $language;
+
+    public function __construct(AttributeRepository $attributeRepository, $language)
     {
         $this->attributeRepository = $attributeRepository;
-        $this->load();
+        $this->language = $language;
     }
 
-    public function load(): void
+    public function getAttributes(): array
```

Comment:
I feel you should reflect in the name of the method, this entails returning a value object specifically used for AttributeType assistance. That way, we are not tempted to use this for other purposes later on.